src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/ProcessNotificationTest.java

```diff
@@ -49,7 +49,7 @@ private boolean testNotification(JsonNode jsonNode, QueueMessageDetails messageD
             .hasSize(1)
             .first()
             .satisfies(node -> {
-                assertThat(node.get("confirmation_id").asText()).isNotEmpty();
+                assertThat(node.get("confirmation_id").asText()).isBlank();
```

Comment:
worth adding a todo or comment here that this is temporary so that we will remember in future

Reply:
need to change `status` as well ;)

Reply:
PENDING I guess